.github/workflows/docs.yml

```diff
@@ -0,0 +1,55 @@
+name: Documentation CI
+on:
+  pull_request:
+    paths:
+      - .vale.ini
+      - docs/**/*.md
+      - Library/Homebrew/**/*.rb
```

Comment:
Is this because these could be inputs here? This feels like pretty much everything so I wonder if it's worth restricting the paths here at all.

Note also if we restrict paths we can't make this a required build.

Reply:
Ah, so in other words: for a workflow to be a merge requirement, it needs to run on every PR.

Reply:
@EricFromCanada Exactly!